### DISCUSSION TITLE
Exceptions that occur during copying a backup are not forwarded to the top.

### DISCUSSION THREAD

**Discussion by Manuuu93** [spatie/laravel-backup#1407] [Q&A] [answered]
Exceptions in the copyToBackupDestinations method are handled but not thrown up. Isn't such an error a backup error in general?
https://github.com/spatie/laravel-backup/blob/eddb4ad167c2371fd321e9b414268cc1a683170b/src/Tasks/Backup/BackupJob.php#L277
As a result, we get the message "Backup completed" but this is not so.

**Comment by erikn69** [comment, marked as answer]
As there can be several disks, the exception is caught and continues with the next one, "Backup completed" means that the command ended
